girderformindlogger/web_client/src/models/UserModel.js

```diff
@@ -122,6 +122,24 @@ var UserModel = AccessControlledModel.extend({
         });
     },
 
+    /**
+     * Change the login for this user.
+     */
+    updateLogin: function (newLogin) {
+        return restRequest({
+            url: `${this.resourceName}/login`,
+            data: {
+                login: newLogin
+            },
+            method: 'PUT',
+            error: null
+        }).done(() => {
+            this.trigger('g:loginChanged');
+        }).fail((err) => {
+            this.trigger('g:error', err);
+        });
+    },
+
```

Comment:
This is good but unnecessary. This interface is vestigial from the upstream Girder project and not one in use for MindLogger.